styles/site-styles.css

```diff
@@ -0,0 +1,164 @@
+
+@font-face {
+    font-family: gotham-light;
+    src: url(./Gotham-Font/Gotham-Light.otf) url(./Gotham-Font/GothamLight.ttf);
+}
+
+@font-face {
+    font-family: gotham-black;
+    src: url(./Gotham-Font/Gotham-Black.otf) url(./Gotham-Font/GothamBold.ttf);
+}
+
+body {
+    margin: auto;
+    color: white;
+    background-color: rgb(216,192,208);
+    font-family: gotham-light;
+}
+
+header {
+    display: flex;
+    height: 96px;
```

Comment:
you also set the height of the header in `styles.css` and in your html, it is last in the cascade so that size `60px` gets chosen.

Reply:
I actually did this one on purpose because the site I'm attempting to replicate had a taller header on every other page but the landing page. However, I do think it's a silly choice, so I might change it in the end. I just wanted to try to get to get it as close as possible at first to challenge myself.